airbyte-integrations/connectors/source-google-analytics-data-api/source_google_analytics_data_api/source.py

```diff
@@ -243,6 +259,11 @@ def request_body_json(
         next_page_token: Mapping[str, Any] = None,
     ) -> Optional[Mapping]:
         payload = super().request_body_json(stream_state, stream_slice, next_page_token)
+
+        # remove offset and limit fields according to their absence in
+        # https://developers.google.com/analytics/devguides/reporting/data/v1/rest/v1beta/properties/runPivotReport
+        payload.pop("offset")
+        payload.pop("limit")
```

Comment:
To make this operation safe to not break in case the key doesn't exist.
```suggestion
        payload.pop("offset", None)
        payload.pop("limit", None)
```

Reply:
done :heavy_check_mark: